lib/enrichers/schema.relationships.enricher.js

```diff
@@ -95,19 +104,46 @@ function enrichSchemaWithRelationships(
 function _resolveModelProperties(
   modelClass,
   isIncludeRelationships,
-  isIncludeParentRelationships
+  isIncludeParentRelationships,
+  modelPaths
 ) {
-  const model = _.isString(modelClass) ? require(modelClass) : modelClass;
+  let model;
+
+  if ( _.isString(modelClass)) {
+    if (modelPaths && modelPaths.length) {
+      for (const modelPath of modelPaths) {
+        let fullPath = path.join(process.cwd(), modelPath, modelClass);
+
+        if (!fullPath.endsWith('.js'))
+          fullPath += '.js';
+
+        if (fs.existsSync(fullPath)) {
+          model = require(fullPath);
+
+          break;
```

Comment:
can all or most of the new code block be moved into an internal function? then instead of this logic with `break` it would just do an early return of a value.

Reply:
sure

Reply:
Done